test/CountBase.test.ts

```diff
@@ -19,8 +21,22 @@ describe('CountBase baselet', function () {
         }
       }
     })
-    const disklet = makeMemoryDisklet()
-    await createCountBase(disklet, dbName, dbBucketSize)
+    countbaseDb = await createCountBase(disklet, dbName, dbBucketSize)
     expect(await disklet.getText(`${dbName}/config.json`)).equals(expectedTest)
   })
+  it('insert data', async function () {
```

Comment:
Ideally, the unit tests should actually use the insert and query functions instead of trying to read the Disklet directly

Reply:
I was trying to keep the "insert data" and "query data" unit tests independent of each other. In order to do that I needed a third-party method to verify. But I can put them together and just have "insert and query data" test.

Reply:
OK. Totally fair. As long as we are testing both sides